ansible/roles/docker/tasks/install-fedora-28.yml

```diff
@@ -0,0 +1,42 @@
+- name: 'Ensure "imposter" versions of Docker are gone'
```

Comment:
I'm going to be a bit nitpicky – can we remove the single quotes from all of these strings, please?

Reply:
I don't know if I like that style more. I think this makes it clearer that the entire thing is a string.

Reply:
I dislike the inconsistency, and personally prefer unquoted strings in YAML, but I'd take quoting them all too.